LayoutTests/imported/w3c/web-platform-tests/css/css-images/image-set/image-set-parsing-expected.txt

```diff
@@ -77,10 +77,10 @@ PASS e.style['background-image'] = "image-set(url(\"example.png\") 0dpi)" should
 PASS e.style['background-image'] = "-webkit-image-set(url(\"example.png\") 0dpi)" should set the property value
 PASS e.style['background-image'] = "image-set(url(\"example.png\") 0dpcm)" should set the property value
 PASS e.style['background-image'] = "-webkit-image-set(url(\"example.png\") 0dpcm)" should set the property value
-FAIL e.style['background-image'] = "image-set(url(\"example.png\") calc(-1 * 1x))" should set the property value assert_equals: serialization should be canonical expected "image-set(url(\"example.png\") calc(-1 * 1x))" but got "image-set(url(\"example.png\") calc(-1x))"
-FAIL e.style['background-image'] = "-webkit-image-set(url(\"example.png\") calc(-1 * 1x))" should set the property value assert_equals: serialization should be canonical expected "image-set(url(\"example.png\") calc(-1 * 1x))" but got "image-set(url(\"example.png\") calc(-1x))"
-FAIL e.style['background-image'] = "image-set(url(\"example.png\") calc(1x + -1x))" should set the property value assert_equals: serialization should be canonical expected "image-set(url(\"example.png\") calc(1x + -1x))" but got "image-set(url(\"example.png\") calc(0x))"
-FAIL e.style['background-image'] = "-webkit-image-set(url(\"example.png\") calc(1x + -1x))" should set the property value assert_equals: serialization should be canonical expected "image-set(url(\"example.png\") calc(1x + -1x))" but got "image-set(url(\"example.png\") calc(0x))"
+FAIL e.style['background-image'] = "image-set(url(\"example.png\") calc(-1 * 1x))" should set the property value assert_equals: serialization should be canonical expected "image-set(url(\"example.png\") calc(-1 * 1x))" but got "image-set(url(\"example.png\") calc(-1dppx))"
+FAIL e.style['background-image'] = "-webkit-image-set(url(\"example.png\") calc(-1 * 1x))" should set the property value assert_equals: serialization should be canonical expected "image-set(url(\"example.png\") calc(-1 * 1x))" but got "image-set(url(\"example.png\") calc(-1dppx))"
+FAIL e.style['background-image'] = "image-set(url(\"example.png\") calc(1x + -1x))" should set the property value assert_equals: serialization should be canonical expected "image-set(url(\"example.png\") calc(1x + -1x))" but got "image-set(url(\"example.png\") calc(0dppx))"
+FAIL e.style['background-image'] = "-webkit-image-set(url(\"example.png\") calc(1x + -1x))" should set the property value assert_equals: serialization should be canonical expected "image-set(url(\"example.png\") calc(1x + -1x))" but got "image-set(url(\"example.png\") calc(0dppx))"
```

Comment:
These should now pass if you want to resync that specific test from upstream: https://github.com/web-platform-tests/wpt/blob/ce1b8fa687/css/css-images/image-set/image-set-parsing.html

Reply:
ah, I was trying to figure out why it failed and didn't realize it needs a sync as well. thanks for catching it! I will handle it in this PR. 🙏  (for `css/mediaqueries`, I will handle it in https://bugs.webkit.org/show_bug.cgi?id=258438)

Reply:
Imported the tests in the new commit.